clang/test/SemaSYCL/reqd-sub-group-size-host.cpp

```diff
@@ -1,9 +1,52 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-host -fsyntax-only -verify %s
-// expected-no-diagnostics
+// RUN: %clang_cc1 -fsycl -fsycl-is-host -Wno-sycl-2017-compat -ast-dump %s | FileCheck %s
```

Comment:
You still need - verify in RUN line to verify no diagnostics are emitted

Reply:
Yes, agree. I missed that. Added now.